services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/IndividualRequestMdbTest.java

```diff
@@ -1,70 +1,53 @@
- * - Products derived from this software may not be called "Money Claims OnLine" nor may
- * "Money Claims OnLine" appear in their names without prior written permission of the
- * Ministry of Justice.
- * - Redistributions of any form whatsoever must retain the following acknowledgment: "This
- * product includes Money Claims OnLine."
- * This software is provided "as is" and any expressed or implied warranties, including, but
- * not limited to, the implied warranties of merchantability and fitness for a particular purpose are
- * disclaimed. In no event shall the Ministry of Justice or its contributors be liable for any
- * direct, indirect, incidental, special, exemplary, or consequential damages (including, but
- * not limited to, procurement of substitute goods or services; loss of use, data, or profits;
- * or business interruption). However caused any on any theory of liability, whether in contract,
- * strict liability, or tort (including negligence or otherwise) arising in any way out of the use of this
- * software, even if advised of the possibility of such damage.
- *
- * $Id: $
- * $LastChangedRevision: $
- * $LastChangedDate: $
- * $LastChangedBy: $ */
 package uk.gov.moj.sdt.services.messaging;
 
-import org.easymock.EasyMock;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.moj.sdt.services.TargetApplicationSubmissionService;
+
 import uk.gov.moj.sdt.services.messaging.api.ISdtMessage;
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 import java.io.Serializable;
 import javax.jms.JMSException;
 import javax.jms.Message;
 import javax.jms.ObjectMessage;
 
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.*;
+
 /**
  * Test class for Message reader implementation.
  *
  * @author Manoj Kulkarni
  */
+@ExtendWith(MockitoExtension.class)
 public class IndividualRequestMdbTest extends AbstractSdtUnitTestBase {
     /**
      * Mock service for the message processing.
      */
+    @Mock
     private TargetApplicationSubmissionService mockTargetSubmissionService;
 
     /**
      * Individual Request Mdb Test instance.
      */
     private IndividualRequestMdb individualRequestMdb;
 
+    private static final String TEST_SUCCESSFULLY_COMPLETED = "Test successfully completed";
+
     /**
      * Pre-testing initialization.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        mockTargetSubmissionService = EasyMock.createMock(TargetApplicationSubmissionService.class);
+        mockTargetSubmissionService = mock(TargetApplicationSubmissionService.class);
```

Comment:
This mock will be initialised by the Mock annotation so it doesn't need to be done here.  The whole method can be removed if it isn't needed.

Reply:
Done